features/manuals_publisher.feature

```diff
@@ -0,0 +1,8 @@
+@app-manuals-publisher
+Feature: Manuals Publisher
+
+  @local-network
+  Scenario: Healthcheck
+    Given I am testing "manuals publisher" internally
+    When I request "/healthcheck"
+    Then I should get a 200 status code
```

Comment:
Looks like this needs a newline at the end of the file (I think that's the convention, though can't find a doc to back that up 🤔 )

Reply:
I think I fixed this as you were typing this comment haha